modules/nf-core/custom/tabulartogseagct/meta.yml

```diff
@@ -0,0 +1,37 @@
+name: "custom_tabulartogseagct"
+description: Convert a TSV or CSV with features by row and observations by column to a GCT format file as consumed by GSEA
+keywords:
+  - gsea
+  - gct
+tools:
+  - "tabulartogseagct":
+      description: "Convert a TSV or CSV with features by row and observations by column to a GCT format file as consumed by GSEA"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing matrix information
+        e.g. [ id:'test' ]
+  - tsv:
+      type: file
+      description: TSV or CSV file
```

Comment:
Given this is very specific module for a very specific tool, I would make this more specific than just 'any' TSV/CSV file